src/helpers.js

```diff
@@ -3,30 +3,50 @@
 import FS from 'sb-fs'
 import Path from 'path'
 import semver from 'semver'
-import { exec } from 'sb-exec'
+import { BufferedProcess } from 'atom'
 import type { Dependency } from './types'
 
 let shownStorageInfo = false
 const VALID_TICKS = new Set(['✓', 'done'])
 const VALIDATION_REGEXP = /(?:Installing|Moving) (.*?) to .* (.*)/
 
+export function exec(command: string, parameters: Array<string>): Promise<{ stdout: string, stderr: string }> {
```

Comment:
Making it async adds extra clutter when it's transpiled. I would have added an async if I was using await inside, anywhere